requirements.txt

```diff
@@ -1,7 +1,8 @@
 mako==1.0.7
 edx-i18n-tools==0.4.7
 django-statici18n==1.8.0
+six==1.12.0
 transifex-client==0.13.4
-git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
-git+https://github.com/edx/XBlock.git@xblock-0.5.0#egg=XBlock==0.5.0
+git+https://github.com/edx/xblock-utils.git@master#egg=master
+git+https://github.com/edx/XBlock.git@xblock-1.2.4#egg=XBlock==1.2.4
```

Comment:
0.5.0 is very old release and latest release is python3 compatible.

Reply:
You can pullthis from PyPI now instead of github. https://pypi.org/project/XBlock/